src/AbstractInput.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Input\Http;
+
+use Yiisoft\Hydrator\Validator\ValidatedInputInterface;
+use Yiisoft\Hydrator\Validator\ValidatedInputTrait;
+
+abstract class AbstractInput implements RequestInputInterface, ValidatedInputInterface
+{
+    use ValidatedInputTrait;
```

Comment:
Isn't it easy enough to add this line instead of `extends AbstractInput`?

Reply:
```php
final class EditPostInput implements RequestInputInterface, ValidatedInputInterface
{
    use ValidatedInputTrait;
    // ...
```

===
```php
final class EditPostInput extends AbstractInput
{
    // ...
```

Reply:
`AbstractInput` is wrapper for  `RequestInputInterface`, `ValidatedInputInterface` and `ValidatedInputTrait` only.